src/request.ts

```diff
@@ -1186,6 +1187,48 @@ class DatastoreRequest {
     this.save(entities, callback);
   }
 
+  merge(entities: Entities): Promise<CommitResponse>;
+  merge(entities: Entities, callback: CallOptions): void;
+  /**
+   * Maps to {@link Datastore#save}, forcing the method to be `merge`.
```

Comment:
Can you beef up the docs for this method? It should be mentioned how the process works and how it is accomplished.